lib/dry/system/stubs.rb

```diff
@@ -0,0 +1,22 @@
+module Dry
+  module System
+    class Container
+      # Incuded only in the Test environment
+      # Sending the message enable_stubs! allow you to stub components after
+      # finalize your container in your tests.
+      #
+      # @api private
+      module Stubs
+        def finalize!(&block)
+          super(freeze: false, &block)
+        end
+      end
+
+      class << self
+        def enable_stubs!
+          extend ::Dry::System::Container::Stubs
+        end
+      end
```

Comment:
Gah I meant that you can drop `class << self` and simply do `def self.enable_stubs!` which does the extend.